quizbizz/src/util/toasts/Toasts.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import { useToast } from "../../context/toast-context";
+import { AiFillCloseSquare } from "../../assets/icons/icons";
+import "./toasts.css";
+
+function Toasts() {
+  const { toastList, dispatchToast } = useToast();
+
+  const deleteToast = (id) => {
+    let newToastList = toastList.filter((toast) => toast.id !== id);
+
+    dispatchToast({
+      type: "SET_LIST",
+      payload: newToastList,
+    });
+  };
+
+  useEffect(() => {
+    let timeoutId;
+    if (toastList.length > 0) {
+      timeoutId = setTimeout(() => {
+        dispatchToast({
+          type: "SET_LIST",
+          payload: [],
+        });
+      }, 3000);
+    }
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, [toastList]);
+
+  const getToastByType = (type) => {
+    switch (type) {
+      case "SUCCESS":
```

Comment:
Suggestion: you could have just made the toast type class by writing
```
`toast-${type.toLowerCase}`
```

Reply:
Ya but i like it this way, this seems more readable to me